tests/fixtures/sample_lib/blocks/other_task/other_implementation.py

```diff
@@ -16,7 +19,7 @@ def __init__(self, batch_size=64, learning_rate=0.0015):
         self.batch_size = batch_size
         self.learning_rate = learning_rate
 
-    def run(self, sample_input: SampleInputType) -> OtherOutputType:
+    def run(self, sample_input: SampleInputType) -> Union[OtherOutputType, str]:
```

Comment:
super good enough for ow, definitely need some dang tests to follow on the `serializers` that are now no longer serializers